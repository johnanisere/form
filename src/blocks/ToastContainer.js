import styled from "styled-components";

const Container = styled.div`
  display: flex;
  align-items: center;
  margin-right: 16px;
`;

export default Container;
