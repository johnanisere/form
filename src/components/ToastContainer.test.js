import React from "react";
import { shallow } from "enzyme";

import ToastContainer from "./ToastContainer";

it("ToastContainer component exists", () => {
  const wrapper = shallow(<ToastContainer />);
  expect(wrapper.exists()).toBe(true);
});

it("ToastContainer component renders", () => {
  const wrapper = shallow(<ToastContainer />);
  expect(wrapper).toMatchSnapshot();
});
