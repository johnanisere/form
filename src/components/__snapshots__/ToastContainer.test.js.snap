// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToastContainer component renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "autoClose": 5000,
      "bodyClassName": null,
      "className": null,
      "closeButton": <InlineSVG
        baseURL=""
        cacheGetRequests={false}
        className="toast-close"
        onLoad={[Function]}
        src="https://res.cloudinary.com/drqqv1c2v/image/upload/v1543619816/Close.svg"
        supportTest={[Function]}
        uniquifyIDs={true}
        wrapper={[Function]}
      />,
      "closeOnClick": true,
      "draggable": true,
      "draggablePercent": 80,
      "hideProgressBar": true,
      "newestOnTop": false,
      "pauseOnFocusLoss": true,
      "pauseOnHover": true,
      "position": "top-right",
      "progressClassName": null,
      "progressStyle": null,
      "rtl": false,
      "style": null,
      "toastClassName": null,
      "transition": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "autoClose": 5000,
        "bodyClassName": null,
        "className": null,
        "closeButton": <InlineSVG
          baseURL=""
          cacheGetRequests={false}
          className="toast-close"
          onLoad={[Function]}
          src="https://res.cloudinary.com/drqqv1c2v/image/upload/v1543619816/Close.svg"
          supportTest={[Function]}
          uniquifyIDs={true}
          wrapper={[Function]}
        />,
        "closeOnClick": true,
        "draggable": true,
        "draggablePercent": 80,
        "hideProgressBar": true,
        "newestOnTop": false,
        "pauseOnFocusLoss": true,
        "pauseOnHover": true,
        "position": "top-right",
        "progressClassName": null,
        "progressStyle": null,
        "rtl": false,
        "style": null,
        "toastClassName": null,
        "transition": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
